--- /tmp/tree-checker.c.orig	2026-05-29 11:07:51.164193129 +1000
+++ /home/ubuntu/linux-source-5.3.0/fs/btrfs/tree-checker.c	2026-05-29 03:43:33.679892226 +1000
@@ -817,13 +817,13 @@
 			btrfs_inode_nlink(leaf, iitem));
 		return -EUCLEAN;
 	}
-	if (btrfs_inode_flags(leaf, iitem) & ~BTRFS_INODE_FLAG_MASK) {
-		inode_item_err(fs_info, leaf, slot,
-			       "unknown flags detected: 0x%llx",
-			       btrfs_inode_flags(leaf, iitem) &
-			       ~BTRFS_INODE_FLAG_MASK);
-		return -EUCLEAN;
-	}
+//	if (btrfs_inode_flags(leaf, iitem) & ~BTRFS_INODE_FLAG_MASK) {
+//		inode_item_err(fs_info, leaf, slot,
+//			       "unknown flags detected: 0x%llx",
+//			       btrfs_inode_flags(leaf, iitem) &
+//			       ~BTRFS_INODE_FLAG_MASK);
+//		return -EUCLEAN;
+//	}
 	return 0;
 }
 
@@ -907,12 +907,12 @@
 	}
 
 	/* Flags check */
-	if (btrfs_root_flags(&ri) & ~valid_root_flags) {
-		generic_err(leaf, slot,
-			    "invalid root flags, have 0x%llx expect mask 0x%llx",
-			    btrfs_root_flags(&ri), valid_root_flags);
-		return -EUCLEAN;
-	}
+//	if (btrfs_root_flags(&ri) & ~valid_root_flags) {
+//		generic_err(leaf, slot,
+//			    "invalid root flags, have 0x%llx expect mask 0x%llx",
+//			    btrfs_root_flags(&ri), valid_root_flags);
+//		return -EUCLEAN;
+//	}
 	return 0;
 }
 
